generators/server/templates/_README.md

```diff
@@ -296,13 +296,13 @@ For more information, refer to the [Running tests page][].
 ## Using Docker to simplify development (optional)
 
 You can use Docker to improve your JHipster development experience. A number of docker-compose configuration are available in the [src/main/docker](src/main/docker) folder to launch required third party services.
-For example, to start a <%= prodDatabaseType%> database in a docker container, run:
+For example, to start the JHipster Registry in a docker container, run:
```

Comment:
This is unrelated change why?

Reply:
Quite simply because when the 'no database' option is selected, the following doc is generated that makes no sense: 

For example, to start a **no** database in a docker container, run:
docker-compose -f src/main/docker/**no**.yml up -d

Reply:
Same way Registry is not always present so if you choose no registry it won't make sense as well and no registry is more common then no database

Reply:
@deepu105  May be better than displaying 'No database' and 'docker/no.yml'

So we can add then a condition to test if database is selected. In this case we display the example for database otherwise we display the example forJHipster Registry or Consul as generally 'No database' option is selected in Microservices architecture (gateway or microservice app).

What do you think about that? if you agree I change and I commit.

Reply:
Yes condition is better, if no DB selected then no need to show the section at all